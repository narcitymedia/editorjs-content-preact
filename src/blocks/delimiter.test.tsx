import React from 'react';
import * as renderer from 'react-test-renderer';
import { BlockType } from '..';

import Delimiter from './delimiter';

describe('Delimeter', () => {
  it('should render the delimeter', () => {
    const tree = renderer
      .create(<Delimiter type={BlockType.Delimiter} data={null} contentUniqueId="XXX-XXX" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
