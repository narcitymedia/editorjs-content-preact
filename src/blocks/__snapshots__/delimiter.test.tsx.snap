// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delimeter should render the delimeter 1`] = `
<hr
  id="XXX-XXX"
/>
`;
